tests/pipelines/test_pipelines_text_generation.py

```diff
@@ -240,7 +240,11 @@ def run_pipeline_test(self, text_generator, _):
         # We don't care about infinite range models.
         # They already work.
         # Skip this test for XGLM, since it uses sinusoidal positional embeddings which are resized on-the-fly.
-        if tokenizer.model_max_length < 10000 and "XGLM" not in tokenizer.__class__.__name__:
+        EXTRA_MODELS_CAN_HANDLE_LONG_INPUTS = ["RwkvForCausalLM", "XGLMForCausalLM"]
```

Comment:
I didn't go into the detail in `Rwkv` modeling, but long inputs works (at least here). However, it has `tokenizer.model_max_length < 1024`. Not sure if we want to change that value.